src/actions/index.js

```diff
@@ -29,6 +32,19 @@ export const getAllTransactions = () => dispatch => {
 		});
 };
 
+export const clearTransactions = () => {
+	return({
+		type: types.CLEAR_TRANSACTIONS,
+	});
+};
+
 export const enableTransactionPolling = () => dispatch => {
 	setInterval(() => dispatch(getAllTransactions()), 5000);
 };
+
+export const setWalletAddress = newWalletAddressValue => {
```

Comment:
Can we wrap this with `checkFSA`, and also `newWalletAddressValue` -> `walletAddress` - we know it's new and that it's a value

Reply:
Changed to `updateSettings`.  - Wrapped with `checkFSA`